rebasehelper/cli.py

```diff
@@ -90,14 +90,15 @@ def build_parser():
             action="store_true",
             help="be more verbose (recommended)"
         )
-        parser.add_argument(
+        patch_build_only_group = parser.add_mutually_exclusive_group()
```

Comment:
Why call this variable `patch_build_only_group` while a little bit lower similar variable is called simply `group`?